app/views/pages/_contributor_blurb.html.erb

```diff
@@ -0,0 +1,34 @@
+<div class="gridManyItem center">
```

Comment:
Would love to tune up the responsive behavior of this component. 😄

When the viewport is <= 640px, I like that you made the modal fill the entire screen, but it would also be super cool if the avatar sat on its own line (centered, I think), the name, location, and link were centered, and the bio kept left-aligned.